smartmin/csv_imports/models.py

```diff
@@ -8,8 +10,16 @@
 from smartmin.models import SmartModel
 
 
+def generate_file_path(instance, filename):
```

Comment:
@rowanseymour I verified this is safe django always makes sure the file names are unique by appending 8 characters to the file name